rip/core.py

```diff
@@ -878,21 +879,43 @@ def prompt_creds(self, source: str):
         :type source: str
         """
         if source == "qobuz":
-            secho("Enter Qobuz email:", fg="green")
-            self.config.file[source]["email"] = input()
-            secho(
-                "Enter Qobuz password (will not show on screen):",
-                fg="green",
-            )
-            self.config.file[source]["password"] = md5(
-                getpass(prompt="").encode("utf-8")
-            ).hexdigest()
+            secho("Use Qobuz auth token to authenticate? (yes/no)", fg="green")
```

Comment:
If you type Y/y it would still work. Because of the regex `(?i)^y` which checks if the first letter in the input provided by the user is a "y" and also it is not case-sensitive.

So if user provides
yes
y
Y
YES
ye
... so on

All of the above mean that he wants to use auth token